### PR TITLE
Contact Form validation

### PR DESCRIPTION
**Pull request by samiullahbahadur**
# We Create a sample Form validation by JavaScript

1. I used style attribute [here](https://github.com/samiullahbahadur/Portfolio/blob/26cc49d584ae84ff3a37e4e582f0f2c2bcd241b4/style.css#L830-L839)
2. I used JavaScript technic for validation form [here](https://github.com/samiullahbahadur/Portfolio/blob/26cc49d584ae84ff3a37e4e582f0f2c2bcd241b4/js/script.js#L136-L166)

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -151,7 +155,7 @@ const success = () => {
   error.classList.add('deactive-error');
 };
 
-submitBtn.addEventListener('click', (event) => {
+submitBtn.addEventListener('click', event => {
   const pattern = /[A-Z]/;
   if (pattern.test(emailInpute.value)) {
     failed(event);
```
Comment (medaminedev66) [inline js/script.js]: - [ ] The form validation still doesn't work. I submitted a form with an email `ex: Amine@gmail.com` and the form was submitted successfully which It's supposed to show an error next to the submit button. Kindly check the requirements again:
- The contact form contains at least one validation checking if the email is written in `all lowercase`:
  - If a user submits the contact form with the email field containing uppercase characters an error message should show up.
  - If a user submits the contact form with the email field containing only lowercase characters the form is sent normally.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @samiullahbahadur , :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_See the inline comments below for more required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elyor-doniyorov — APPROVED** [review]
Hi @samiullahbahadur, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/script.js (hunk 1 above)
Comment (samiullahbahadur) [reply js/script.js]: yes, you right.
I'm fix the error

**Pull request merged**